package.json

```diff
@@ -1,8 +1,29 @@
 {
   "name": "website",
+  "scripts": {
+    "_build": "hugo --cleanDestinationDir -e dev -DFE",
+    "_get:submodule": "set -x && git submodule update --init ${DEPTH:- --depth 1}",
+    "_prebuild": "npm run get:submodule",
+    "_prepare:docsy": "cd themes/docsy && npm install",
+    "_serve:hugo": "hugo serve -DFE --minify",
+    "build:preview": "set -x && npm run _build -- --minify --baseURL \"${DEPLOY_PRIME_URL:-/}\"",
+    "build:production": "hugo --cleanDestinationDir --minify",
+    "build": "npm run _build",
+    "get:submodule": "npm run _get:${GET:-submodule}",
+    "postget:submodule": "git submodule",
+    "prepare": "npm run get:submodule && npm run _prepare:docsy",
+    "serve": "npm run _serve:hugo",
+    "test": "echo 'TODO: setup link checking'"
+  },
   "devDependencies": {
     "autoprefixer": "^10.4.0",
-    "postcss": "^8.4.1",
+    "hugo-extended": "^0.119.0",
```

Comment:
@chalin are you sure that this is the correct way to set the Hugo version on Netelify?

Because the Netelify website suggests using `HUGO_VERSION`, which is what we were previously doing.

Reply:
This is what we're using on other sites such as OTel: https://github.com/open-telemetry/opentelemetry.io/blob/299ad7bb24488e39eb17d43393dd05f513321253/package.json#L89

The advantage of using `hugo-extended` is that you'll never have to worry again about using the right version of Hugo even if (at some point in the future) you work on branches of older versions of the docs since NPM install and the NPM build commands will naturally use the version of Hugo needed for that specific branch/commit.